misk/src/main/kotlin/misk/web/interceptors/TracingInterceptor.kt

```diff
@@ -29,7 +32,21 @@ internal class TracingInterceptor internal constructor(private val tracer: Trace
   }
 
   override fun intercept(chain: NetworkChain): Response<*> {
-    val scope = tracer.buildSpan("${chain.action}").startActive(true)
+    var parentContext: SpanContext? = null
+    try {
+      parentContext = tracer.extract(Format.Builtin.HTTP_HEADERS,
+          TextMultimapExtractAdapter(chain.request.headers.toMultimap()))
+    } catch (exception: Exception) {
+      logger.info("Failure attempting to extract span context. Existing context, if any," +
```

Comment:
What can cause this failure? Should this be at info level logging?

Reply:
Corrupted trace data (e.g. mangled header) can cause a failure. Thinking about it, we probably want to bump this to warn. Should be rare enough and people probably want to know.